doc/ug/getting_started_dv.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Getting Started with an OpenTitan Design Verification"
+---
+
+This document aims to enable a contributor to get started with a design verification (DV) effort within the OpenTitan project.
+While most of the focus is on development of a testbench from scratch, it should also be useful to understand how to contribute to an existing effort.
+Please refer to the [DV methodology]({{< relref "dv_methodology.md" >}}) document for information on how design verification is done in OpenTitan.
+
+## Stages of DV
+
+The life stages of a design / DV effort within the OpenTitan are described in the [Hardware Development Stages]({{< relref "doc/project/hw_stages.md" >}}) document.
+It separates the life of DV into three broad stages: Initial Work, Under Test and Testing Complete.
+This document attempts to give guidance on how to get going with the first stage and have a smooth transition into the Under Test stage.
+They are not hard and fast rules but methods we have seen work well in the project.
+DV indeed cannot begin until the design has transitioned from Specification to the Development stage first.
+The design specification, once available, is used as a starting point.
+
+## Getting Started
+
+The very first thing to do in any DV effort is to [document the plan]({{< relref "dv_methodology.md#documentation" >}}) detailing the overall effort.
+This is done in conjunction with developing the initial testbench.
+It is recommended to use the [uvmdvgen]({{< relref "util/uvmdvgen/README.md" >}}) tool, which serves both needs.
+
+The `uvmdvgen` tool provides the ability to generate the outputs in a specific directory.
+This should be set to the root of the DUT directory where the `rtl` directory exists.
+When the tool is run, it creates a `dv` directory, along with `data` and `doc` directories.
+The `dv` directory is where the complete testbench along with the collaterals to build and run tests can be found.
+It puts the documentation sources in `doc` and `data` directories respectively (which also exist alongside the `rtl` directory).
+It is recommended to grep for 'TODOs' at this stage in all of these generated files to make some of the required fixes right way.
```

Comment:
nit: grep for 'TODO', perhaps remove the 's'

Reply:
Good catch, thanks.